packages/design-picker/src/types.ts

```diff
@@ -21,6 +21,8 @@ export interface DesignRecipe {
 
 export type DesignFeatures = 'anchorfm'; // For additional features, = 'anchorfm' | 'feature2' | 'feature3'
 
+export type DesignType = 'vertical' | 'premium' | 'standard' | 'default' | 'anchor-fm'; // For measuring what kind of the design user picked
```

Comment:
Or maybe `free` instead of `standard` makes more sense as I think `premium` also belongs to `standard` 🤔 cc @autumnfjeld 

I'll add comments to describe the meaning of `default`